python/requirements/development.txt

```diff
@@ -18,3 +18,4 @@ pyparsing
 pysam
 PyVCF
 python_jsonschema_objects
+biopython
```

Comment:
I can't see biopython used anywhere in the PR - so is this needed? OTOH, I may be missing something.

Reply:
I'm using it in the ``tests/test_parsimony.py`` to compare the Sankoff distances we compute with the Biopython version. It's a good sanity check.

Reply:
Ah - github just didn't load test_parsimony in my web browser window, so I didn't find it on initial search